libs/utils/android/benchmark.py

```diff
@@ -116,7 +116,7 @@ def _parseCommandLine(self):
 
         # Test customization
         parser.add_argument('--results-dir', type=str,
-                default=None,
+                default=self.__class__.__name__,
```

Comment:
It's a sensible default but it's a misalignment - benchmarks use  `$LISA_HOME/results/<CLASS_NAME>` while synthetics etc use `$LISA_HOME/results/<DATE_TIME>`